lib/test/cdo/test_profanity_filter.rb

```diff
@@ -59,4 +59,12 @@ def test_find_potential_profanities_locale
 
     WebPurify.unstub(:find_potential_profanities)
   end
+
+  def test_find_potential_profanities_replace_text_list
```

Comment:
This test might benefit from a comment or maybe slightly clearer example text to make it more readable, because it took me a while to wrap my brain around it!

Reply:
Good point.  Updated.